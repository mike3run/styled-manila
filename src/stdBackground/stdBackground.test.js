import React from 'react'
import styled from 'styled-components'
import renderer from 'react-test-renderer'
import 'jest-styled-components'
import { stdBackground } from '..'

const Background = styled.div`
  ${stdBackground}
`

test('it works', () => {
  const tree = renderer.create(<Background />).toJSON()
  expect(tree).toMatchSnapshot()
  expect(tree).toHaveStyleRule('background-position', 'center')
  expect(tree).toHaveStyleRule('background-size', 'cover')
  expect(tree).toHaveStyleRule('background-repeat', 'no-repeat')
})
