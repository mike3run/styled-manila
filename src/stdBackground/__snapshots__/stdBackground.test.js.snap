// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it works 1`] = `
.c0 {
  background-position: center;
  background-size: cover;
  background-repeat: no-repeat;
}

<div
  className="c0"
/>
`;
